dsc/workflows/base/__init__.py

```diff
@@ -590,7 +590,7 @@ def finalize_items(self) -> None:
                 )
             )
 
-    def process_result_messages(self) -> list[str]:
+    def process_result_messages(self) -> None:
```

Comment:
Not sure if I fully grasp why this is being changed. Originally, I thought DynamoDB would just be a part of `Wiley` workflow-specific processing but now that every workflow has it, the best example would be the DDC workflow where an ArchivesSpace URI will be included in the metadata but not put into the DSpace metadata. It would carry over though (I'm just now thinking maybe through DynamoDB?) so that a report can be produced of ArchivesSpace URIs and their corresponding DSpace handles after `finalize` is run. This report would be used in a separate process from DSC to update ArchivesSpace

Reply:
Makes sense, and will do on updating the Jira ticket!